app/assets/javascripts/salaries.js

```diff
@@ -27,20 +28,28 @@ function draw_salaries_chart() {
   employees.map(function(employee) {
     data_table.addColumn({
       "type": "number",
-      "label": employee.display_name + " " + employee.display_pay
+      "label": employee.display_name
+    });
+    data_table.addColumn({
+      "type": "string",
+      "role": "tooltip"
     });
   });
 
   data.map(function(row) {
+    const salary_date = new Date(row[0]);
     var rest = row.slice(1);
-    var rest_to_add = rest.map(function(x) {
+    var rest_to_add = rest.flatMap(function(x, i) {
       if(x !== null) {
-        return {v: parseFloat(x)};
+        return [{v: parseFloat(x)}, {
```

Comment:
While we're updating this file, unless the single-letter variable names are required for our chart tools it might make sense to use more descriptive ones. (The index variable is probably fine, but v in particular is a little confusing.)

Reply:
I do like that. Will do

Reply:
Can you also fix v?

Reply:
I don't believe so. That is the "v" property for the data object.

https://developers.google.com/chart/interactive/docs/reference#dataparam

Reply:
Ha! Unless we drop it entirely